src/shared/components/query/QueryStore.ts

```diff
@@ -1278,8 +1278,18 @@ export class QueryStore
 			const caseSetId =  (window as any).serverVars.caseSetProperties.case_set_id;
 			if (caseSetId !== undefined) {
 				this.selectedSampleListId = caseSetId;
+				this.initiallySelected.sampleListId = true;

```

Comment:
how come we need to remember initially selected distinct from selectedSampleListId?  could use a comment